content/ja/docs/concepts/workloads/controllers/job.md

```diff
@@ -0,0 +1,888 @@
+<!--
+reviewers:
+- erictune
+- soltysh
+title: Jobs
+content_type: concept
+feature:
+  title: Batch execution
+  description: >
+    In addition to services, Kubernetes can manage your batch and CI workloads, replacing containers that fail, if desired.
+weight: 50
+-->
+---
+reviewers:
+title: Jobs
+content_type: concept
+feature:
+  title: バッチ実行
+  description: >
+    サービスだけでなく、KubernetesはバッチとCIワークロードの管理機能も提供し、必要に応じて障害が発生したコンテナを引き換えることもできます。
+weight: 50
+---
+
+<!-- overview -->
+
+<!--
+A Job creates one or more Pods and will continue to retry execution of the Pods until a specified number of them successfully terminate. As pods successfully complete, the Job tracks the successful completions.  When a specified number of successful completions is reached, the task (ie, Job) is complete.  Deleting a Job will clean up the Pods it created. Suspending a Job will delete its active Pods until the Job is resumed again.
+-->
+Jobは一つ以上のPodを作成し、指定された数のPodが正常に終了するまで、Podの実行を再試行し続けます。Podが正常に終了すると、Jobは成功したPodの数を追跡します。指定された完成数に達すると、タスク（つまりJob）の完成となります。Jobを削除すると、作成されたPodも一緒に削除されます。Jobを一時停止すると、再開されるまで、稼働しているPodは全部削除されます。
+
+<!--
+A simple case is to create one Job object in order to reliably run one Pod to completion.
+The Job object will start a new Pod if the first Pod fails or is deleted (for example due to a node hardware failure or a node reboot).
+-->
+単純なケースを言うと、一つのPodが正常に完成するまで実行することを確保するため、一つのJobオブジェクトを作成します。
+一つ目のPodに障害が発生したり、削除されたり（例えばノードのハードウェア障害またノードの再起動が起こした場合）すると、Jobオブジェクトは新しいPodを作成してくれます。
+
+<!--
+You can also use a Job to run multiple Pods in parallel.
+-->
+Jobで複数のPodを並行して実行することもできます。
+
+<!--
+If you want to run a Job (either a single task, or several in parallel) on a schedule, see[CronJob](/docs concepts/workloads/controllers/cron-jobs/).
+-->
+スケジュールに沿ってJob（単一のタスクか複数タスク並行のいずれか）を実行したい場合は [CronJob](https://kubernetes.io/ja/docs/concepts/workloads/controllers/cron-jobs/)を参照してください。
```

Comment:
In addition to suggestions by @nasa9084, we could update link URL in same manner (ie. it does not include kubernetes.io domain).
```suggestion
スケジュールに沿ってJob（単一のタスクか複数タスク並行のいずれか）を実行したい場合は [CronJob](/ja/docs/concepts/workloads/controllers/cron-jobs/)を参照してください。
```

Reply:
Could you check this type of update on other part in this PR?